docs/external_authentication/README.md

```diff
@@ -0,0 +1,51 @@
+External Authentication Documentation
+============
+
+This README describes how to create a one-machine installation of all the components necessary for testing LDAP authentication in MongoDB.
+
+# Overview
+
+Normally, a client needs to authenticate themselves against the internal MongoDB user database before doing any work or reading any data from a `mongod` or `mongos` instance.  This project aims to allow the client to authenticate themselves through the MongoDB server to an independent service, such as OpenLDAP or Active Directory.
```

Comment:
Not sure, but perhaps "*through* the MongoDB server *to* an independent service" should be replaced with "*to* the MongoDB server *through* an independent service".